src/instructlab/model/chat.py

```diff
@@ -18,11 +20,16 @@
 from rich.markdown import Markdown
 from rich.panel import Panel
 from rich.text import Text
+import click
 import httpx
 import openai
 
+# First Party
+from instructlab import client, config
+from instructlab.config import DEFAULT_CONNECTION_TIMEOUT, DEFAULT_MODEL_OLD
+from instructlab.server import is_temp_server_running
```

Comment:
This is minor but this could be moved into the function level import in `def chat()` since it's only used there